community/iproute2/build

```diff
@@ -0,0 +1,13 @@
+#!/bin/sh -e
+
+patch -p1 < 0001-make-iproute2-fhs-compliant.patch
+
+./configure \
+    --prefix=/usr \
+    --sbindir=/usr/bin
```

Comment:
These don't actually work. Default prefix is `/usr` and you've probably installed these to `/usr/sbin`!